src/types/Setting.ts

```diff
@@ -333,12 +333,15 @@ export interface CarConnectorConnectionSetting {
   description: string;
   timestamp: Date;
   type: CarConnectorConnectionType;
+  token?: CarConnectorConnectionToken;
   mercedesConnection?: CarConnectorMercedesConnectionType;
```

Comment:
I would expect 'mercedes' and 'tronity'.
We have a link between the Type and the name of the prop:
![2021-09-22_19-37-04](https://user-images.githubusercontent.com/25266945/134393739-7accbc8a-9da2-48cf-9aeb-e589a5e7e17b.png)

And we use it when updating the Tenant's components in the mater tenant.

Reply:
The whole car connector framework is built like the asset framework.

If we choose the connection at master tenant level, we are not able to use different connectors for different vehicles, which is required, when we have vehicles of different manufacturers in our fleet.

Reply:
My bad, the mercedesConnector prop is one level below so that's okay.